code/game/machinery/cryopod.dm

```diff
@@ -702,6 +702,10 @@
 	allow_occupant_types = list(/mob/living/silicon/robot)
 	disallow_occupant_types = list(/mob/living/silicon/robot/drone)
 
+/obj/machinery/cryopod/robot/offstation
+	//For offstation cyborgs
+	silent = TRUE
+
```

Comment:
When you cryo as a cyborg, it does not show up on the local Robotic Storage Console (neither on the on-station one):

![image](https://user-images.githubusercontent.com/33333517/180659427-cb46b57f-df96-4bef-b726-34c36e6dc582.png)

...

It shows up in the Cryogenic Oversight Console instead:

![image](https://user-images.githubusercontent.com/33333517/180659441-97dcbc0d-8e72-44fd-8bab-afadf11edac4.png)